PIL/Image.py

```diff
@@ -1554,7 +1554,7 @@ def resize(self, size, resample=NEAREST):
 
         return self._new(self.im.resize(size, resample))
 
-    def rotate(self, angle, resample=NEAREST, expand=0):
+    def rotate(self, angle, center=None, translate=None, resample=NEAREST, expand=0):
```

Comment:
Should `center` and `translate` be added as last parameters, in case existing code already calls `rotate` assuming the third and fourth parameters are `resample` and `expand`?